amical/tools.py

```diff
@@ -302,11 +343,10 @@ def apply_windowing(img, window=80, m=3):
     distance = np.sqrt(xx2**2 + yy2[:, np.newaxis] ** 2)
 
     # Super-gaussian windowing
-    window = super_gaussian(distance, sigma=window * 2, m=m)
+    super_gauss = super_gaussian(distance, window=window, m=m)
```

Comment:
```suggestion
    super_gauss = super_gaussian(distance, sigma=window, m=m)
```

Reply:
Fixed it! Also in `data_processing` where the `super_gaussian` function was called.